### PR TITLE
Bepis cola, for that expensive taste that pops!

### PR DESCRIPTION
**Pull request by ArcaneMusic**
## About The Pull Request
image

Kids, there's nothing more cool than slamming down a refreshing bepis brand cola. Ahh, that nice fizzly taste that goes well with a nice Nanotrasen hotdog, and way, WAY better than that knock off Space Cola. Did you hear that they make that with rats? I heard they make it out of distilled rat. It's Rat cola. Drink Bepis.

## Why It's Good For The Game

I have lost all control

## Changelog
:cl:
add: The hottest, COLDEST drink around!
del: Your abillity to drink all INFERIOR sodas
/:cl:

### CODE REVIEW

File: code/modules/vending/_vending.dm [hunk 1]
```diff
@@ -797,6 +797,9 @@ GLOBAL_LIST_EMPTY(vending_products)
 					vend_ready = TRUE
 					return
 				var/datum/bank_account/D = SSeconomy.get_dep_account(payment_department)
+				if(istype(R.product_path,/obj/item/reagent_containers/food/drinks/soda_cans/cola))
+					SSeconomy.get_dep_account(ACCOUNT_SCI).adjust_money(price_to_use)
+					D = null
```
Comment (Qustinnus) [inline code/modules/vending/_vending.dm]: this does not do what you think it does

**Comment by HomocidlePancakes** [comment]
This seems fairly normal. Why is it an april fools PR?

**Comment by BobJoga** [comment]
>it's the same reagent as space cola
>doesn't add space cola hater trait
-1 bad

**Comment by cacogen** [comment]
Pepsi is cheaper than Coke and tastes worse. Ergo, Bepis should be cheaper than Space Cola and tastes worse.

**Comment by ArcaneMusic** [comment]
Sorry gang, bepis bottling co bought the exclusive distribution rights for soft drinks in the sector, nothin I can do here

**Comment by cacogen** [comment]
I'm depressed and ashamed

**Pull request closed without merging**